portal/gatsby-ssr.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
```

Comment:
Gatsby-ssr gjør at vi kan akksesere det som normalt ville vært en index.html, som ligger under the hood i gastby. Tilsvarende gatsby-browser, men da også på server side.